### PR TITLE
Update links in the README

### PR DESCRIPTION
**Pull request by GomathiselviS**
This PR corrects some of the incorrect links and updates all links to relative ones.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,35 +17,35 @@ Click on the name of a role, playbook, or rulebook to view that content's docume
...
-[cloud.aws_ops.clone_on_prem_vm](https://github.com/ansible-collections/cloud.aws_ops/blob/main/roles/clone_on_prem_vm/README.md)|A role to clone an existing on prem VM using the KVM hypervisor.
-[cloud.aws_ops.import_image_and_run_aws_instance](https://github.com/ansible-collections/cloud.aws_ops/blob/main/roles/import_image_and_run_aws_instance/README.md)|A role that imports a local .raw image into an Amazon Machine Image (AMI) and run an AWS EC2 instance.
+[cloud.aws_ops.aws_setup_credentials](roles/aws_setup_credentials/README.md)|A role to define credentials for aws modules.
+[cloud.aws_ops.awsconfig_detach_and_delete_internet_gateway](roles/awsconfig_detach_and_delete_internet_gateway/README.md)|A role to detach and delete the internet gateway you specify from virtual private cloud.
+[cloud.aws_ops.awsconfig_multiregion_cloudtrail](roles/awsconfig_multiregion_cloudtrail/README.md)|A role to create/delete a Trail for multiple regions.
+[cloud.backup_create_plan](roles/backup_create_plan/README.md)|A role to create an AWS backup plan.
+[cloud.backup_select_resources](roles/backup_create_plan/README.md)|A role to select resources to back up with an existing backup plan.
```
Comment (mandar242) [inline README.md:24]:
```suggestion
[cloud.backup_select_resources](roles/backup_select_resources/README.md)|A role to select resources to back up with an existing backup plan.
```

**Review by mandar242 — APPROVED** [review]

File: playbooks/move_vm_from_on_prem_to_aws/README.md [hunk 2]
```diff
@@ -4,12 +4,12 @@ A playbook to migrate an existing on prem VM running on KVM hypervisor to AWS.
 
 ## Requirements
 
-This playbook uses the ``cloud.aws_ops.clone_on_prem_vm`` role to clone an existing VM on prem using the KVM hypervisor and the ``cloud.aws_ops.import_image_and_run_aws_instance`` role to import a local .raw image into an Amazon machine image (AMI) and run an AWS EC2 instance. For a complete list of requirements, see [clone_on_prem_vm](../clone_on_prem_vm/README.md#Requirements) and [import_image_and_run_aws_instance](../roles/import_image_and_run_aws_instance/REAME.md#Requirements), respectively.
+This playbook uses the ``cloud.aws_ops.clone_on_prem_vm`` role to clone an existing VM on prem using the KVM hypervisor and the ``cloud.aws_ops.import_image_and_run_aws_instance`` role to import a local .raw image into an Amazon machine image (AMI) and run an AWS EC2 instance. For a complete list of requirements, see [clone_on_prem_vm](../../roles/clone_on_prem_vm/README.md#Requirements) and [import_image_and_run_aws_instance](../../roles/import_image_and_run_aws_instance/REAME.md#Requirements), respectively.
```
Comment (hakbailey) [inline playbooks/move_vm_from_on_prem_to_aws/README.md:7]:
```suggestion
This playbook uses the ``cloud.aws_ops.clone_on_prem_vm`` role to clone an existing VM on prem using the KVM hypervisor and the ``cloud.aws_ops.import_image_and_run_aws_instance`` role to import a local .raw image into an Amazon machine image (AMI) and run an AWS EC2 instance. For a complete list of requirements, see [clone_on_prem_vm](../../roles/clone_on_prem_vm/README.md#Requirements) and [import_image_and_run_aws_instance](../../roles/import_image_and_run_aws_instance/README.md#Requirements), respectively.
```

File: playbooks/move_vm_from_on_prem_to_aws/README.md [hunk 3]
```diff
@@ -4,12 +4,12 @@ A playbook to migrate an existing on prem VM running on KVM hypervisor to AWS.
...
 
 
 ## Playbook Variables
 
-For a full list of accepted variables see: [clone_on_prem_vm](../clone_on_prem_vm/README.md#Role-Variables) and respectively [import_image_and_run_aws_instance](../roles/import_image_and_run_aws_instance/REAME.md#Role-Variables).
+For a full list of accepted variables see: [clone_on_prem_vm](../../roles/clone_on_prem_vm/README.md#Role-Variables) and respectively [import_image_and_run_aws_instance](../../roles/import_image_and_run_aws_instance/REAME.md#Role-Variables).
```
Comment (hakbailey) [inline playbooks/move_vm_from_on_prem_to_aws/README.md:12]:
```suggestion
For a full list of accepted variables see: [clone_on_prem_vm](../../roles/clone_on_prem_vm/README.md#Role-Variables) and respectively [import_image_and_run_aws_instance](../../roles/import_image_and_run_aws_instance/README.md#Role-Variables).
```

File: roles/awsconfig_apigateway_with_lambda_integration/README.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ AWS User Account with permission to create API gateway, lambda function and IAM
 
 GNU General Public License v3.0 or later
 
-See [LICENCE](https://github.com/ansible-collections/cloud.aws_ops/blob/main/LICENSE) to see the full text.
+See [LICENCE](../../LICENSE) to see the full text.
```
Comment (hakbailey) [inline roles/awsconfig_apigateway_with_lambda_integration/README.md:45]:
```suggestion
See [LICENSE](../../LICENSE) to see the full text.
```

Comment (hakbailey) [reply roles/awsconfig_apigateway_with_lambda_integration/README.md:45]: There are a bunch more cases of this typo below that we should probably update.

**Review by hakbailey — CHANGES_REQUESTED** [review]

**Review by alinabuzachis — APPROVED** [review]

**Comment by alinabuzachis** [comment]
Unrelated to this PR, but the integration tests fails with:

```
fatal: [testhost]: FAILED! => {
    "msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'arn'\n\nThe error appears to be in '/home/runner/collections/ansible_collections/cloud/aws_ops/roles/backup_select_resources/tasks/main.yaml': line 54, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n    - name: Set backup role ARN\n      ^ here\n"
}
```

Would you like to have a look in this PR @GomathiselviS? The fix might be trivial.

**Comment by GomathiselviS** [comment]
> Unrelated to this PR, but the integration tests fail with:
> 
> ```
> fatal: [testhost]: FAILED! => {
>     "msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'arn'\n\nThe error appears to be in '/home/runner/collections/ansible_collections/cloud/aws_ops/roles/backup_select_resources/tasks/main.yaml': line 54, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n    - name: Set backup role ARN\n      ^ here\n"
> }
> ```
> 
> Would you like to have a look in this PR @GomathiselviS? The fix may be trivial.

I will work on this in a separate PR. Thank you for pointing out the failure.

**Pull request merged**